Controller/ApiController.php

```diff
@@ -40,9 +36,6 @@ class ApiController
     /** @DI\Inject("jms_translation.config_factory") */
     private $configFactory;
 
-    /** @DI\Inject */
-    private $request;
-
```

Comment:
So I feel like this should be in a separate PR. The removal of the use statements above as well. They are unrelated to the changes and could easily be their own PR and merged quickly. I made similar changes in an unpushed PR with lots of other Phpdoc block and code cleanup changes.